ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-ospf.j2

```diff
@@ -47,6 +47,26 @@
 {%         set mpls_ldp_sync_default = router_ospf.process_ids[process_id].mpls_ldp_sync_default | arista.avd.default('-') %}
 | {{ process_id }} | {{ router_id }} | {{ passive_interface_default }} |{{ no_passive_interfaces.list }} | {{ bgp_enable }} | {{ max_lsa }} | {{ default_information_originate }} | {{ log_adjacency_changes_detail }} | {{ auto_cost_reference_bandwidth }} | {{ maximum_paths }} | {{ mpls_ldp_sync_default }} |
 {%     endfor %}
+{# OSPF Distance #}
+{%     set ospf_distance_process_ids = [] %}
+{%     for process_id in router_ospf.process_ids %}
+{%         if router_ospf.process_ids[process_id].distance is arista.avd.defined %}
```

Comment:
I was referring to this check.

Reply:
the check in documentation is just to detect if the table should be printed at all.

Reply:
Yeah, I added this comment to refer other comment. Anyways LGTM!